### DISCUSSION TITLE
Will Schwab have an API?

### DISCUSSION THREAD

**Discussion by philgoetz** [alexgolec/tda-api#339] [Q&A]
Any idea what sort of automated trading API Charles Schwab has, or if this API will still work after the transition to Schwab in November?

**Comment by philgoetz** [comment]
TDAmeritrade wrote a message to me, saying:

As part of the Schwab integration, a retail Trader API will be made available at Schwab.  The goal is to provide an updated version of our current API capabilities in Schwab's ecosystem using Schwab's authentication and account support systems. We do not yet have an exact date on when this will be available or what exactly it will look like.

**Reply by arcmode** [reply]
You can signup already at https://beta-developer.schwab.com/

**Comment by philgoetz** [comment]
Did you successfully sign up?  It tells me there was a problem, and I
should try later.
Also, the Terms and Conditions you have to certify having read is just a
blank page.

**Comment by zhagndongjian** [comment]
> Did you successfully sign up?  It tells me there was a problem, and I
> should try later.
> Also, the Terms and Conditions you have to certify having read is just a
> blank page.
> 
> On Sat, Oct 14, 2023 at 3:34 PM David Rojas Camaggi <
> ***@***.***> wrote:
> 
> > You can signup already at https://beta-developer.schwab.com/
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/alexgolec/tda-api/discussions/339#discussioncomment-7282243>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AAVE7BHFPVBMZ2IBVN3QOOTX7LSNFAVCNFSM6AAAAAA4VNQZLKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TEOBSGI2DG>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> 

I have successfully sign up, but there is no information inside.

**Reply by stratx21** [reply]
Any further developments on this? I'm considering transferring back to TDA or moving brokerages to get an API. So far the others don't match how good TDA's was though

**Comment by zhagndongjian** [comment]
I have been standing still




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alexgolec/tda-api"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月10日(星期六) 中午1:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alexgolec/tda-api] Will Schwab have an API? (Discussion #339)





 
Any further developments on this? I'm considering transferring back to TDA or moving brokerages to get an API. So far the others don't match how good TDA's was though

**Comment by philgoetz** [comment]
What do you mean, transferring back to TDA?  How is that possible?

**Reply by stratx21** [reply]
You can still make an account with TDA according to their website, but they have a condition that the account will be moved to Schwab on I think May 10-13 2024

**Comment by zhagndongjian** [comment]
The account has been opened, and my concern is whether the tda-api can continue to be used in the future.Will Schwab have an API?&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alexgolec/tda-api"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月13日(星期二) 凌晨1:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alexgolec/tda-api] Will Schwab have an API? (Discussion #339)





 
You can still make an account with TDA according to their website, but they have a condition that the account will be moved to Schwab on I think May 10-13 2024

**Comment by 48n116w** [comment]
From personal experience, Schwab is currently approving API access to certain endpoints. Currently limited to 'Accounts and Trading' . No market data yet but promised 'soon'. There's significant changes to authorization process.

**Comment by zhagndongjian** [comment]
Will our TDA-api be updated？



---Original---
From: ***@***.***&gt;
Date: Fri, Mar 29, 2024 09:02 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alexgolec/tda-api] Will Schwab have an API? (Discussion #339)




 
From personal experience, Schwab is currently approving API access to certain endpoints. Currently limited to 'Accounts and Trading' . No market data yet but promised 'soon'. There's significant changes to authorization process.
